.github/workflows/update_tf_compatibility_matrix.yml

```diff
@@ -30,4 +30,4 @@ jobs:
           commit-message: "doc: Updates Terraform Compatibility Matrix documentation"
           delete-branch: true
           branch: terraform-compatibility-matrix-update
-          body: Automatic updates for Terraform Compatibility Matrix documentation
+          body: Automatic updates for Terraform Compatibility Matrix documentation. Update .tools-version file and TF_VERSION_LATEST GitHub environment variable if needed.
```

Comment:
I think .tool-versions update can be automated right? [CLOUDP-243394](https://jira.mongodb.org/browse/CLOUDP-243394)

Reply:
it can, and also probably the env var, but as it's not done yet, I would keep the message.  Once it's updates the message can be changed